src/composables/transactionTokens.ts

```diff
@@ -65,11 +64,14 @@ export function useTransactionTokens({
         ...otherToken,
       }));
     }
-    if (transaction.protocol === PROTOCOL_BITCOIN) {
+
+    if (transaction.protocol && transaction.protocol !== PROTOCOL_AETERNITY) {
+      const protocolAdapter = ProtocolAdapterFactory
+        .getAdapter(transaction.protocol);
       return [{
         ...innerTx.value || {},
-        amount: getBitcoinTxAmountTotal(transaction, direction === TX_DIRECTION.received),
-        symbol: BTC_SYMBOL,
+        symbol: protocolAdapter.getCoinSymbol(false),
```

Comment:
1st argument should not be required in my opinion. So `false` should be the default value.